### PR TITLE
CB2-2097: Added test station interfaces

### PR DESCRIPTION
**Pull request by ghost**
## VTA to source ATF information from Dynamics
as part of this work we need new interfaces to hold and transform data
https://jira.dvsacloud.uk/browse/CB2-2097

## Checklist
- [x] Code has been tested manually
- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Squashed commit contains the JIRA ticket number

### CODE REVIEW

**Comment by ghost** [comment]
@premghinde do we need to follow the PR guide into our own feature branches if there's no sub tasks for them really ?

File: src/Interfaces/DynamicsTestStation.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+interface DynamicsTestStation {
+  accountid: string
```
Comment (andrewf-bjss) [inline src/Interfaces/DynamicsTestStation.ts]:
```suggestion
  accountid:string
```

File: src/Interfaces/DynamoTestStation.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+interface DynamoTestStation {
+  testStationId : string,
```
Comment (andrewf-bjss) [inline src/Interfaces/DynamoTestStation.ts]:
```suggestion
  testStationId: string,
```

File: src/Interfaces/DynamoTestStation.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  testStationAddress: string,
```
Comment (andrewf-bjss) [inline src/Interfaces/DynamoTestStation.ts]: I think we're missing `testStationAccessNotes`...

```suggestion
  testStationId : string,
  testStationAccessNotes: string,
  testStationAddress: string,
```

Comment (ghost) [reply src/Interfaces/DynamoTestStation.ts]: Its empty so I'm wondering if dynamo will fill it in as null if its not there

File: src/Interfaces/DynamoTestStation.ts (hunk 2 above)
Comment (ghost) [reply src/Interfaces/DynamoTestStation.ts]: Thanks, need to turn my linter back on

Comment (Ian-Hodges) [reply src/Interfaces/DynamoTestStation.ts]: I was thinking that the linter would have spotted this.

**Review by andrewf-bjss — APPROVED** [review]

File: src/Interfaces/DynamoTestStation.ts (hunk 2 above)
Comment (ghost) [reply src/Interfaces/DynamoTestStation.ts]: I disabled the plugin to look at the other project

**Review by Ian-Hodges — APPROVED** [review]

**Comment by danielchildsbjss** [comment]
> @premghinde do we need to follow the PR guide into our own feature branches if there's no sub tasks for them really ?

I'd be happy to relax the PR guide as it's going to kick in when we go feature -> develop anyway

**Review by danielchildsbjss — APPROVED** [review]

**Comment by premghinde** [comment]
Agree with Dan. Happy to defer paperwork until the final PR into develop.

**Pull request merged**